src/machines/java/com/enderio/machines/common/io/energy/IMachineEnergyStorage.java

```diff
@@ -33,6 +33,11 @@ public interface IMachineEnergyStorage extends IEnergyStorage {
      */
     int consumeEnergy(int energy);
 
+    /**
+     * Can consume energy from storage.
+     * @return If the amount can be taken from the storage.
+     */
+    boolean canConsumeEnergy(int energy);
```

Comment:
Maybe instead of this, mutation operations should have a simulate boolean akin to Forge's energy API?

Reply:
This can be done as this method isn't used anywhere else. Should I make the change ?

Reply:
I'd say so, aye please

Reply:
Done!